### PR TITLE
148-Focus-Command

### LINKED ISSUE #148: New `focus` command
Create a new `focus` command to show more details of the Candidate in the new panel.

### PR DESCRIPTION
**Pull request by tiewweijian**
Currently the middle panel is not used for any commands. 

Let's create a new command called `Focus Command` to bring Candidate's information up to the middle panel ! 

closes #148

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -463,6 +463,27 @@ As the user will not be able to remember the link to our User Guide, we hyperlin
...
+### Focus Feature
+
+#### What is this feature about?
+The 'focus' feature utilises the different panels in our application. It will bring up the intended Candidate's information, 
+such as course, email into the middle panel for the user to see. 
+
+#### How is this feature implemented?
+This feature is implemented with the similar implementation of `CandidateListCard`, as we utilise the JavaFX codes to 
+create the `view`. We also introduced new features such as `ImageView`, where the extra information will consist of a display
+picture for the Candidate (WIP).
```
Comment (charmainehly) [inline docs/DeveloperGuide.md:475]: created a placeholder issue for working on how the display picture will be uploaded (if relevant)! #182

File: src/main/java/seedu/address/ui/FocusCard.java [hunk 2]
```diff
@@ -37,21 +64,42 @@
...
         this.candidate = candidate;
+        id.setText(candidate.getStudentId().toString());
+        name.setText(candidate.getName().fullName);
+        phone.setText(candidate.getPhone().value);
+        email.setText(candidate.getEmail().value);
+        course.setText(candidate.getCourse().course + ", " + candidate.getSeniority().seniority);
+        displayPicture.setImage(new Image(new FileInputStream(BLANK_PICTURE_PATH)));
+        setApplicationStatus(candidate.getApplicationStatus());
+        setInterviewStatus(candidate.getInterviewStatus());
+        setAvailableDays(candidate.getAvailability());
```
Comment (charmainehly) [inline src/main/java/seedu/address/ui/FocusCard.java:98]: If we are adding the confirmed interview slot in the future to the `FocusCard`, can link that future PR to #70 !

**Review by charmainehly — APPROVED** [review]
LGTM!!! Thanks for the making the modifications - the panel looks good!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tiewweijian) [reply docs/DeveloperGuide.md:475]: Oki !! Thanks Charmaine !

**Review by LeeEnEn — APPROVED** [review]
Understandable that the codecov drops. Good job regardless! Merging in!

**Pull request merged**